client/src/pages/WeeklyTable.css

```diff
@@ -28,8 +28,8 @@ th {
 	margin-right: 0;
 	width: 100%;
 	margin-top: 20px;
-	background-color: #ddd4d2;
-	color: black;
+	background-color: #f8f3f2;
```

Comment:
I think this looks a bit weird in the end :/ so I wouldn't include these changes

Reply:
I mean this color and background color in the weeklytable

Reply:
I think both were great before, also more readable